### DISCUSSION TITLE
Actividad 0 - Daza, Facundo

### DISCUSSION THREAD

**Discussion by Shelldaza** [frlp-utn-ingsoft/catedra#25] [Show and tell]
https://github.com/Shelldaza/Actividad0-Ing.software

**Comment by RodrigoJacznik** [comment]
Buenas,
La app funciona muy bien. Los mensajes de los commits también están bien.
El tp esta aprobado.

**Comment by Shelldaza** [comment]
Genial muchas gracias profe
